docs/spec/v1beta2/buckets.md

```diff
@@ -432,6 +433,36 @@ data:
   accountKey: <BASE64>
 ```
 
+##### Azure Blob SAS Token example
+
+```yaml
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: Bucket
+metadata:
+  name: azure-sas-token
+  namespace: default
+spec:
+  interval: 5m0s
+  provider: azure
+  bucketName: <bucket-name>
+  endpoint: https://<account-name>.blob.core.windows.net
+  secretRef:
+    name: azure-key
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: azure-key
+  namespace: default
+type: Opaque
+data:
+  sasKey: <base64> # The SAS token starts with a leading question mark i.e "?spr="
```

Comment:
Given that query parameters are not order sensitive, I am not sure whether we want to force users to always start with `?spr=`.
The same goes to the leading question mark. If the base URL already has some parameters - for whatever reason - we should just add the new ones.

Reply:
Currently, we only expect the leading question mark. The `spr` was an example. I will remove it if it is misleading